diracx/values.yaml

```
@@ -83,6 +84,53 @@ diracx:
   # Which DiracX OpenSearch DBs are used?
   osDatabases:
     - JobParametersDB
+  ingress:
+    enabled: true
+    className: "nginx"
+    annotations:
+      # kubernetes.io/ingress.class: nginx
+      # kubernetes.io/tls-acme: "true"
+      # Needed if you want to use a base path different from /
+      # #Use rewrite rules https://kubernetes.github.io/ingress-nginx/examples/rewrite/
+      nginx.ingress.kubernetes.io/use-regex: "true"
+      nginx.ingress.kubernetes.io/rewrite-target: /$2
+    tls: []
+      #  - secretName: chart-example-tls
+      #    hosts:
+      #      - chart-example.local
+    hosts:
+    - paths:
+        - path: /api(/|$)(.*)
+          pathType: ImplementationSpecific
```

Using `rewrite-target` does not make sense as we are now using `root_path`.
In my mind it works like this :

```
http://ip:8000/api/docs -> nginx ingress rewrite-target -> http://ip:8000/docs -> fastAPI proxy -> http://ip:8000/docs -> fail
```

But in reality, it works fine. I don't get how it can work though.

I wanted to modify these values such as:

```suggestion
    tls: []
      #  - secretName: chart-example-tls
      #    hosts:
      #      - chart-example.local
    hosts:
    - paths:
        - path: /api
          pathType: Prefix
```

And this is actually failing, `/api/docs` or `/api/.well-known/openid-configuration` become unavailable.